src/main/java/github/grace000/erversay/Handlers/Handler.java

```diff
@@ -0,0 +1,8 @@
+package github.grace000.erversay.Handlers;
+
+import github.grace000.erversay.Request;
+
+public interface Handler {
```

Comment:
This is a really generic name. What's it handling?

Reply:
I'm handling the different incoming paths. I'm assigning paths the handlers based on the name of the path. Then the handlers build different responses based on the incoming method that belongs to that path. I'll rename from handler to route handler.